xcc/job.py

```diff
@@ -364,6 +337,64 @@ def __repr__(self) -> str:
         """Returns a printable representation of a job."""
         return f"<{self.__class__.__name__}: id={self.id}>"
 
+    def get_result(
+        self,
+        integer_overflow_protection: bool = True,
+    ) -> Mapping[str, Union[np.ndarray, List[np.ndarray]]]:
+        """Returns the result of a job.
+
+        Args:
+            integer_overflow_protection (bool): convert all NumPy integers into
+                64-bit NumPy integers to avoid surprises during post-processing;
+                setting this option to ``False`` can significantly reduce memory
+                consumption
```

Comment:
What exactly do we mean by "avoid surprises"? Would be better to be explicit about the potential side effects of switching this off
```suggestion
            integer_overflow_protection (bool): convert all NumPy integers into
                64-bit NumPy integers during post-processing; setting this option 
                to ``False`` can significantly reduce memory consumption at the
                risk of <insert consequences>.
```

Reply:
Nice, I like the idea!